dashdotdb/services/deploymentvalidation.py

```diff
@@ -34,15 +36,16 @@ def _insert(self, item):
             self.log.error('skipping validation: key "value" not found')
             return
 
-        expire = datetime.now() - timedelta(minutes=50)
-        db_token = db.session.query(Token) \
-            .filter(Token.timestamp > expire).first()
+        db_token = db.session.query(Tokens) \
+            .filter(Tokens.uuid == token).first()
         if db_token is None:
-            db.session.add(Token(timestamp=datetime.now()))
-            db.session.commit()
-            self.log.info('token created')
-        db_token = db.session.query(Token) \
-            .filter(Token.timestamp > expire).first()
+            self.log.error(f'skipping validation: token not found: {token}')
```

Comment:
what does this return to the client? I guess it should return a 4xx, but I'm unsure which one

Reply:
I believe the existing code would return 200 in this case.  I agree we should be returning error codes in these failures if we can.

Reply:
Since it is in a loop, I believe it will just return a 200.  The insert is going through all the validations sent to it, so its possible 1 of those inserts would fail for one of the new error cases.  Should we return an error for the whole upload (probably)? What code?

Reply:
If token does not exist this must return an error. As discussed above 404 should be fine (I cannot think anythin else). I would always return the same message.

Reply:
Done

Reply:
This will return a 404